### PR TITLE
fixes issue with event sources sidebar to show sink service

### PR DESCRIPTION
**Pull request by invincibleJai**
- fixes issue with event sources sidebar to show sink service

Tracks: https://jira.coreos.com/browse/ODC-2157

ezgif com-video-to-gif (4)

### CODE REVIEW

File: frontend/packages/knative-plugin/src/components/overview/EventSinkServicesOverviewList.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <SidebarSectionHeading text="Knative Services" />
+      {sink && sink.kind === 'Service' ? (
+        <ListGroup componentClass="ul">
+          <li className="list-group-item">
+            <ResourceLink
+              kind={referenceForModel(ServiceModel)}
+              name={sink.name}
+              namespace={namespace}
+            />
+            {sinkUri ? (
```
Comment (debsmita1) [inline frontend/packages/knative-plugin/src/components/overview/EventSinkServicesOverviewList.tsx]: Please use `&&` and remove `null` from `line no. 40`

File: frontend/packages/knative-plugin/src/components/overview/EventSinkServicesOverviewList.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+};
+
+const EventSinkServicesOverviewList: React.FC<EventSinkServicesOverviewListProps> = ({ obj }) => {
+  const sink = _.get(obj, 'spec.sink', null);
+  const sinkUri = _.get(obj, 'status.sinkUri', null);
+  const namespace = _.get(obj, 'metadata.namespace', null);
+  return (
+    <>
+      <SidebarSectionHeading text="Knative Services" />
+      {sink && sink.kind === 'Service' ? (
```
Comment (gijohn) [inline frontend/packages/knative-plugin/src/components/overview/EventSinkServicesOverviewList.tsx]: use `ServiceModel.kind`.

**Comment by spadgett** [comment]
Previous test failure was https://bugzilla.redhat.com/show_bug.cgi?id=1765280

/retest

**Pull request merged**